WMF Framework/CommonStrings.swift

```diff
@@ -337,4 +339,94 @@ public extension CommonStrings {
 
 	static let chineseVariantsAlertDismissButton = WMFLocalizedString("chinese-variants-alert-dismiss-button", value: "No thanks", comment: "Dismiss button on alert used to inform users about Chinese variant support.")
 
+	// Crimean Tatar
+
+	static let crimeanTatarVariantsAlertTitle =  WMFLocalizedString("crimean-tatar-variants-alert-title", value: "Updates to Crimean Tatar variant support", comment: "Title of alert used to inform users about Crimean Tatar variant support.")
+
+	static let crimeanTatarVariantsAlertBody = WMFLocalizedString("crimean-tatar-variants-alert-body", value: "The Wikipedia app now supports the following Crimean Tatar variants as primary or secondary languages within the app, making it easier to read, search and edit in your preferred variants:\n\nQırımtatarca, Latin Crimean Tatar Latin (chr-latn)\nкъырымтатарджа, Кирил Crimean Tatar Cyrillic (crh-cyrl)", comment: "Body text of alert used to inform users about Crimean Tatar variant support. Please do not translate the newlines (\n) or Crimean Tatar characters (къырымтатарджа, etc.).")
+
+	static let crimeanTatarVariantsAlertPreferencesButton = WMFLocalizedString("crimean-tatar-variants-alert-preferences-button", value: "Review your preferences", comment: "Action button on alert used to inform users about Crimean Tatar variant support.")
```

Comment:
@staykids any need to have separate strings for preferences and dismiss? Maybe we could combine them behind two more generic keys, `variants-alert-preferences-button` & `variants-alert-dismiss-button`.

Reply:
Nope - no reason I'm aware of other than mimicking the same pattern I used for Chinese variants. I'll generalize them now. 👍🏽

Reply:
Updated!